### PR TITLE
docs(internalLink): control

### PR DESCRIPTION
**Pull request by CristianSpatari**

### CODE REVIEW

File: docusaurus/docs-internals/editor-controls/data-controls/internalLink.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+Example of the internal link search:
+
+![Internal Link Search](/img/data-controls/internalLinkSearch.png)
+
+Example of the internal link result:
+
+![Internal Link Result](/img/data-controls/internalLinkResult.png)
+
+### Parameters
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/data-controls/internalLink.md:21]: `internalLink` has also the `placeholder` props
image

check if component can render it or how is used and if is needed then add it to parameters table

**Review by victor-pg — CHANGES_REQUESTED** [review]
1. image
2. image
3. image
4. image

**Review by victor-pg — CHANGES_REQUESTED** [review]
1. image
2. image
3. image

**Review by victor-pg — CHANGES_REQUESTED** [review]
1. image
2. image

**Comment by CristianSpatari** [comment]
> 1. image
> 2. image

Aici helper se arunca prin config. Nu stiu ce exemplu sa mai scriu

File: docusaurus/docs-internals/editor-controls/data-controls/internalLink.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+```js
+{
+  value: string;
+  title: string;
+  source: string;
+}
+```
+
+`value`: A string that holds the actual data or content. <br/>
```
Comment (victor-pg) [inline docusaurus/docs-internals/editor-controls/data-controls/internalLink.md:48]: value: the actual selected value
title: the label of the selected value
source: /* here describe that the source is the collection from where we try to find a link */

**Review by victor-pg — CHANGES_REQUESTED** [review]
1. Broken table
image
2. image
3. image

**Review by victor-pg — CHANGES_REQUESTED** [review]
1. image
2. image

**Pull request merged**